decompress_tar.go

```diff
@@ -105,6 +105,10 @@ func untar(input io.Reader, dst, src string, dir bool) error {
 	// We therefore wait until we've extracted everything and then set the mtime and atime attributes
```

Comment:
Can we update the comments to be something like:

```go
// Perform a final pass over extracted directories to update metadata
for ... {
  ...
  // Chmod the directory...
  ...

  // Set the mtime/atime attributes since they would have been changed during extraction
  ...
}
```